packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.html

```diff
@@ -50,6 +50,18 @@ <h2 data-translate="pci_databases_users_add"></h2>
             data-ng-pattern="$ctrl.inputRules.group.pattern"
             data-ng-maxlength="$ctrl.inputRules.group.max"
         />
+        <div class="oui-password__rules" id="username-rules">
```

Comment:
**question:** Why are we using an helper from `oui-password` component here?
Because here, we are relying on a `oui-input` (`type="text"`).
When re-using CSS helpers, we should keep the corresponding meaning.

Reply:
**issue(blocking):** There're some duplicates IDs `id="username-rules"`.

Reply:
Hello @jperchoc, you are still setting duplicates ids here: https://github.com/ovh/manager/commit/42c4e68b7c6b14e29617f1d320727206782f390d with `group-rules` > You shouldn't have twice a same id defined in your HTML template

Reply:
Hi @JeremyDec, you're right, I have the div twice... I fixed it here: https://github.com/ovh/manager/pull/6728/commits/c9bfb2a27c8302b361fc4b9dcd527881d17b6c74
Thank you !